<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Format: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="formatCombobox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Range:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rangeCombobox">
       <item>
        <property name="text">
         <string>Entire Sequence</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In to Out</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="videoGroupbox">
     <property name="title">
      <string>Video</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Codec:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="compressionTypeLabel">
        <property name="text">
         <string>Compression Type:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="compressionTypeCombobox">
        <item>
         <property name="text">
          <string>Quality-based (Constant Rate Factor)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>File size-based (Two-Pass)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="vcodecCombobox"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="heightSpinbox">
        <property name="maximum">
         <number>16777216</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="widthSpinbox">
        <property name="maximum">
         <number>16777216</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Frame Rate:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="videobitrateSpinbox">
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="videoBitrateLabel">
        <property name="text">
         <string>Bitrate (Mbps):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="framerateSpinbox">
        <property name="maximum">
         <double>60.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Advanced</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="audioGroupbox">
     <property name="title">
      <string>Audio</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Codec:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="acodecCombobox"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="samplingRateSpinbox">
        <property name="maximum">
         <number>96000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Sampling Rate:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Bitrate (Kbps/CBR):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="audiobitrateSpinbox">
        <property name="maximum">
         <number>320</number>
        </property>
        <property name="value">
         <number>256</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%p% (ETA: 0:00:00)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renderCancel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>formatCombobox</tabstop>
  <tabstop>videoGroupbox</tabstop>
  <tabstop>vcodecCombobox</tabstop>
  <tabstop>widthSpinbox</tabstop>
  <tabstop>heightSpinbox</tabstop>
  <tabstop>framerateSpinbox</tabstop>
  <tabstop>videobitrateSpinbox</tabstop>
  <tabstop>audioGroupbox</tabstop>
  <tabstop>acodecCombobox</tabstop>
  <tabstop>samplingRateSpinbox</tabstop>
  <tabstop>audiobitrateSpinbox</tabstop>
  <tabstop>renderCancel</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
